src/main/java/com/beantastic/api/controllers/BeanClassController.java

```diff
@@ -17,8 +23,17 @@ public BeanClassController(BeanClassRepository beanClassRepository) {
     }
 
     @GetMapping("/beanClasses")
-    public Iterable<BeanClass> getAllBeanClasses() {
-        return beanClassRepository.findAll();
-    }
+    public ResponseEntity<?> getAllBeanClasses() {
```

Comment:
same comment here as in the enemy pr. I really just thing a get should be consistent. And Bad_Request is an error code for when the user does something wrong.